code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -203,38 +252,53 @@
 	spread_step = 10
 
 /obj/item/projectile/bullet/shotgun/practice
-	name = "practice"
-	damage = 5
-	embed = 0
-
-/* Miscellaneous */
+	name = "practice slug"
+	damage = DAMAGE_SLUG * PRACTICE_MULTIPLIER
```

Comment:
I'm thinking all the practise rounds should have their damage set to zero.

Make them do small amounts of agony isntead, but there's no practical application for making them do actual damage

Reply:
Practice rounds don't serve a real purpose. The only purpose they server separately from rubber, is they don't do significant agony damage. This would allow the person being engaged to not get pain stunned, which would allow for a live fire training exercise. There's just one problem with this, the practice rounds are actually lethal. They don't serve much of any significant role that can't be handled by blank shots, or rubber shots. I personally believe they should be removed entirely, and replaced completely with the likes of the blank round.

Reply:
Will remove the blank shotgun bullet, and make all practice rounds `nodamage = TRUE`.

Reply:
Misread your initial comment, will make the requested change.